pages/common/rsync.md

```diff
@@ -11,7 +11,7 @@
 
 `rsync {{remote_host}}:{{path/to/remote_file}} {{path/to/local_directory}}`
 
-- Transfer any file or directory, in archive (to preserve attributes), resolving contained softlinks, and ignoring already transferred files (unless newer):
+- Transfer any file or directory \[r\], in archive \[a\] to preserve attributes, resolving contained softlinks \[L\], and ignoring already transferred files \[u\] (unless newer):
```

Comment:
I have the same question than @agnivade: has any function the second `a`? I mean, if `a` is "to preserve attributes", this command could just `-Laur` instead of `-Laura`, unless the last `a` means something.

Reply:
I think it's supposed to be a menomic, to make it easy to remember. Why the backslashes here?